gems/sorbet-runtime/test/types/edge_cases.rb

```diff
@@ -196,7 +196,7 @@ def sanity; end
     end
   end
 
-  it 'calls original singleton_method_added when registering hooks' do
+  it "calls a user-defined singleton_method_added when registering hooks" do
     klass = Class.new do
       class << self
         def singleton_method_added(name)
```

Comment:
Do we expect that when written this way the hooks one is called? It doesn't call `super`.

Should we add a test that documents the fact that if the `singleton_method_added` hook doesn't call super, then something else breaks? Or is there actually no problem if the user doesn't call `super`?

Reply:
Oh I see this is what the test you added does.

Reply:
I wonder if we could delete this test, actually. This test seems to be testing for whether or not we call `singleton_method_added` when installing the hooks on a module. In the past, we used to install our `singleton_method_added` hook by using `replace_method`, which would call the existing `singleton_method_added` hook. But now, we don't trigger the `singleton_method_added` hook when installing the hooks because we install the hooks not with `replace_method`, but by having the module include/extend some private modules.

Reply:
Resolved: Let's keep it around in case stuff with the hooks change in the future.